archaius-core/src/main/java/com/netflix/config/DynamicDoubleProperty.java

```diff
@@ -39,4 +39,4 @@ public double get() {
     public Double getValue() {
         return get();
     }
-}
+}
```

Comment:
what're all these format changes? can we get rid of those format changes only files?